src/components/dev-hub/article-share-footer.js

```diff
@@ -28,6 +28,10 @@ const BlogShareLinks = styled('div')`
     a:not(:last-of-type) {
         margin-right: ${size.medium};
     }
+    /* For the tooltip, there is a wrapper span to give a proper tab UX */
+    span > span:first-child {
+        margin-right: 0;
```

Comment:
This is a bit wonky, but we want to make sure the margin applied on line 29 for the footer is not double-applied

Reply:
do you know why we have span on line 27? just inspecting the page I don't really see any reason for it. do we need that? if so, can we just apply those styles directly?

Reply:
Good point! The copy link button is actually a span element so we need to give some space between that and the next link. I actually realized because it is set up this way, we can just simplify the logic to only target `a` tags, which will only target the last three links and give left margin instead